rollup.config.mjs

```diff
@@ -0,0 +1,38 @@
+// @ts-check
+import typescript from '@rollup/plugin-typescript';
+
+/**
+ * @type { import("rollup").OutputOptions }
+ */
+const commonOutput = {
+  sourcemap: true,
+  interop: "esModule",
+}
+
+/**
+ * @type { import("rollup").RollupOptions }
+ */
+const config = {
+  input: 'src/decimal128.mts',
+  output: [
+    {
+      ...commonOutput,
+      file: './dist/decimal128.mjs',
+      format: 'esm',
+    },
+    {
+      ...commonOutput,
+      file: './dist/decimal128.cjs',
+      format: 'cjs',
+    },
+  ],
+  plugins: [
+    typescript({
+      tsconfig: "tsconfig.build.json",
+      outDir: "dist",
+      sourceMap: false,

```

Comment:
####Note to reviewer

Source map is still generated by using rollup's `output.sourcemap: true` above.

This is turned off to prevent conflict with rollup's source map generation
See related stackoverflow post: https://stackoverflow.com/a/63235210